mantidproject/mantidimaging
# Compare Operation Window

Fixes #578 

To Test:
- Load Data
- Open Operations window
- Ensure safe apply is checked
- Ensure that all options in the GUI are working
- Ensure that all ROI and when changing images occurs, it syncs across both stacks.
- Check that normal operations occur like before if safe apply is unchecked
